frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -1,28 +1,46 @@
 <template lang='pug'>
-  modal-template(class='has-background is-centered')
+  modal-template(
+    class='is-centered'
+    :class='{"has-background": !isConfirmation}'
+  )
     template(slot='subtitle')
       i18n New proposal
     template(slot='title')
-      i18n(v-if='isConfirmation') Your proposal was created
-      i18n(v-else) {{ title }}
+      i18n(key='title1' v-if='isConfirmation') Your proposal was created
+      span(v-else) {{ title }}
 
     form
-      transition(name='fade' mode='out-in')
+      transition-group
```

Comment:
@pieer I need your help here to make this transition-group work properly... What am I missing here?

Reply:
Transition group work like transition (except it add a span to wrap the content. But it still need a name for you to add the the transition in css.

Reply:
What it does is basically add class on element that hide and show or in this case on the container. But there is no magic, you still need to implement all the class like

Reply:
.fade-enter-active,
.fade-leave-active

Reply:
I tried to do a good animation on this but it seems too tricky, so I ended up by removing it. Feel free to try it too when you tweak the modals!